src/ImageSharp/Metadata/Profiles/Exif/ExifProfile.cs

```diff
@@ -134,7 +136,7 @@ public IReadOnlyList<IExifValue> Values
     /// <returns>
     /// The <see cref="Image{TPixel}"/>.
     /// </returns>
-    public Image<TPixel> CreateThumbnail<TPixel>()
+    public Image<TPixel>? CreateThumbnail<TPixel>()
```

Comment:
Question if this should throw, or perhaps change it to TryCreateThumbnail?

Reply:
It certainly shouldn't throw as EXIF metadata is not guaranteed to contain a thumbnail. I would use `TryCreate`.